### PR TITLE
add instructions to readme for local env and data setup

### PR DESCRIPTION
**Pull request by lady3bean**
This PR adds instructions for adding environment variables and data from the staging server.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -50,6 +50,19 @@ Run:
   $ bin/setup
   ```
 
+
+### Add Local Environment Variables
+Run `cp .sample.env .env` to create a new .env file
```
Comment (germsvel) [inline README.md]: The `bin/setup` script [already does this](https://github.com/thoughtbot/constable/blob/master/bin/setup#L6-L10), so I'm not sure if it makes sense to call out explicitly. What do you think about removing this first line?

File: README.md [hunk 2]
```diff
@@ -50,6 +50,19 @@ Run:
...
+In `.env`, there are three values from the sample file that say get-from-staging,
+CLIENT_ID, CLIENT_SECRET, and HUB_API_TOKEN. Replace these with the values from
+staging. Find them by running `heroku config -r staging`.
+
+
+### Copy the staging database to your development database
+Run `heroku pg:pull STAGING_DB_ENV_VAR constable_api_development -r staging`,
+where STAGING_DB_ENV_VAR is found from heroku staging. This will copy the data
+over so you have example data to work with.
```
Comment (germsvel) [inline README.md]: What do you think about adding some context as to why you would want to copy some of the database data and set `CLIENT_ID` and `CLIENT_SECRET`? (Namely that they are all needed in order to make google auth work locally)

Now that I'm writing this, I wonder if it makes sense to combine these two new sections into a single section that is titled something like `### Setting up data for google auth`. Not sure. What are your thoughts?

File: README.md (hunk 1 above)
Comment (lady3bean) [reply README.md]: sure!

File: README.md (hunk 2 above)
Comment (lady3bean) [reply README.md]: I like it! Will update.

**Pull request merged**